test/test-versions.js

```diff
@@ -47,6 +48,9 @@ function testVersions(dataFilename) {
               console.error('\x1b[31m  version_added: "' + statement.version_added + '" is not a valid version number when version_removed is present');
               console.error('  Valid', browser, 'versions are:', validBrowserVersions[browser].length > 0 ? 'true, ' + validBrowserVersions[browser].join(', ') : 'true');
               hasErrors = true;
+            } else if (typeof statement.version_added === "string" && typeof statement.version_removed === "string" && compareVersions(statement.version_added, statement.version_removed) == 1) {
```

Comment:
Based upon the documentation of the package, it will only return -1 (less than), 0 (equal to), or 1 (greater than), so doing a check for any value greater than 0 seems a little unnecessary, but ah whatever, right?  😛